RPIPs/RPIP-49.md

```diff
@@ -119,6 +121,7 @@ The tokenomics rework package will likely be split between two protocol upgrades
 * [RPIP-44: Forced exits](RPIP-44.md)
 * [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
   * Heuristic adjustments.
```

Comment:
```suggestion
  * Implementing the [Specification taking effect with Saturn 2](https://github.com/rocket-pool/RPIPs/blob/spring2024-tokenomics/RPIPs/RPIP-46.md#specification-taking-effect-with-saturn-2) section. This section includes (a) removing RPL issuance rewards to NOs. (b) Implementing the revenue share vote, including the `voter_share` adjustment heuristic if a `new_share` is utilized.
```

Reply:
Wrote a small bit instead of linking folks out